### PR TITLE
Improvements to Accessibility

### PR DESCRIPTION
**Pull request by simonmitchell**
Makes sure accessibility hint, accessibility label and accessibility are set on buttons created for the hover bar, based on the passed in UIBarButtonItem's values

### CODE REVIEW

**Comment by simonmitchell** [comment]
@CooperRS any chance of getting this looked at? :D

File: SampleApp/ViewController.m [hunk 1]
```diff
@@ -23,6 +23,9 @@ - (void)viewDidLoad {
     UIButton *infoButton = [UIButton buttonWithType:UIButtonTypeInfoLight];
     [infoButton addTarget:self action:@selector(toggleOrientation:) forControlEvents:UIControlEventTouchUpInside];
     UIBarButtonItem *infoBarButton = [[UIBarButtonItem alloc] initWithCustomView:infoButton];
+    UIBarButtonItem *refreshButton = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"icon-refresh"] style:UIBarButtonItemStylePlain target:nil action:nil];
```
Comment (shagedorn) [inline SampleApp/ViewController.m]: Not sure if I'm missing something, but isn't this entirely unused?

**Review by shagedorn — COMMENTED** [review]
Nice addition, thanks! Just wondering about the refresh item.

File: SampleApp/ViewController.m (hunk 1 above)
Comment (simonmitchell) [reply SampleApp/ViewController.m]: So it is, apologies! Fixing for you now ☺️

Comment (shagedorn) [reply SampleApp/ViewController.m]: …but this item still doesn't do anything – what's the motivation to add it here? Unless it demonstrates something crucial, I would rather leave out the added assets and keep this PR small.

 To me, only the changes to `ISHHoverBar.m` and `ISHHoverBarTests.m` are actually expected 😅

Comment (simonmitchell) [reply SampleApp/ViewController.m]: Ahh gotcha!! This all happened a while ago sorry, I think my plan was to use it as an example for the new properties, but happy to remove it!

File: SampleApp/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {
   "images" : [
+    {
```
Comment (shagedorn) [inline SampleApp/Assets.xcassets/AppIcon.appiconset/Contents.json:3]: I guess that's just what Xcode generates these days 😆

**Review by shagedorn — APPROVED** [review]
Awesome, thanks for the quick follow-ups. Would be great if you could squash it all together before merging (or with a squash&merge) 👍

File: SampleApp/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 2 above)
Comment (simonmitchell) [reply SampleApp/Assets.xcassets/AppIcon.appiconset/Contents.json:3]: Seems that way, sorry!

Comment (shagedorn) [reply SampleApp/Assets.xcassets/AppIcon.appiconset/Contents.json:3]: Don't worry, can stay I think!

**Comment by simonmitchell** [comment]
Squashed! :D

**Comment by shagedorn** [comment]
Thanks for your contribution!

**Comment by simonmitchell** [comment]
You're welcome! ☺️

**Pull request merged**